cli/cmd/plugin/ui/web/tanzu-ui/src/shared/components/Select/Select.tsx

```diff
@@ -0,0 +1,85 @@
+// React imports
+import React, { ChangeEvent } from 'react';
+
+// Library imports
+import { CdsSelect } from '@cds/react/select';
+import { CdsControlMessage } from '@cds/react/forms';
+import { CdsIcon } from '@cds/react/icon';
+import { UseFormRegister } from 'react-hook-form';
+import { CdsProgressCircle } from '@cds/react/progress-circle';
+
+// App imports
+import './Select.scss';
+import Tooltip from '../Tooltip/Tooltip';
+import TooltipContent from '../Tooltip/TooltipContent/TooltipContent';
+
+interface Props {
+    label?: string;
+    className?: string;
+    defaultValue?: string;
+    error?: string;
+    disabled?: boolean;
+    controlMessage?: string;
+    isLoading?: boolean;
+    name: string;
+    tooltipMessage?: string;
+    handleSelect: (event: ChangeEvent<HTMLSelectElement>) => void;
+    children: React.ReactNode;
+    register: UseFormRegister<any>;
+}
+function Select(props: Props) {
```

Comment:
Sorry to have yet another suggestion, but...
When I saw the usage of this (below), it occurred to me that maybe we should try for a more descriptive name than `Select` -- some name that better conveys the special functionality. (`Select` felt like a standard SelectBox.)
Something like `LoadableSelect` or `SpinnerSelect`?

Reply:
+1 good suggestion

Reply:
renamed